config/routes.rb

```diff
@@ -77,7 +77,7 @@ def not_implemented
             constraints: ToolMountPoint.for(:advice_plans)
 
       mount DecisionTrees::Engine => '/tools/:tool_id',
-            constraints: ToolMountPoint.for(:decision_trees)
+            constraints: { tool_id: %r{health-check-questions|workplace-pension-advice-tool} }
```

Comment:
are you sure you don't want to use the mount point but with some implementation changes?
Or if you go down your route, at the very least grab these strings from the mount point?